tests/unit/core/components/TestAMInstallButton.js

```diff
@@ -418,7 +418,7 @@ describe(__filename, () => {
       _tracking,
       addon,
       isExperimentEnabled: false,
-      variant: VARIANT_INCLUDE_WARNING,
+      variant: VARIANT_INCLUDE_WARNING_CURRENT,
```

Comment:
Since there are three tests here that need to enable the experiment but don't need to make assertions about `variant`, they seem like good candidates for a shared render helper.